build/build-extensions.ps1

```diff
@@ -1,14 +1,15 @@
 param (
   [string]$buildNumber = "0",  
-  [string]$suffix = "",  
+  [string]$suffix = "",
+  [string]$minorVersionPrefix = "10",
```

Comment:
This is a big nit... but leave this one default to `""` and let the proj file apply the default. I foresee some confusion in the future if we change the proj file and then gotta figure out where the `10` is coming from.